ZTors.tex

```diff
@@ -1102,6 +1102,78 @@ \subsection{Induction in $\TorZ$}\label{sec:TorZ-induction}
+Thus, it suffices to observe that if $B$ is an object with a point $1\to \mathrm{Contr}(B)$ then $B$ is equivalent to the terminal object (a detailed proof can be found in~\cite[Lemma 4.1]{shulman:elreedy}).
+
+It is also worth saying a few words about why our $\TorZ$ ``classifies $\ZZ$-torsors'' in the $(\infty,1)$-categorical sense, i.e.\ that for any object $A$, the hom-space $\sE(A,\TorZ)$ is naturally equivalent to the $\infty$-groupoid of small $\ZZ$-torsors in $\sE/A$.
+Importantly, the relevant notion of ``$\ZZ$-torsor'' is the topos- and sheaf-theoretic one: a $\ZZ$-torsor $X$ in $\sE/A$ need not be ``globally'' isomorphic to $\ZZ\times A$, but only ``locally'' so in the sense that there is an effective epimorphism $p:B \twoheadrightarrow A$ such that $p^*X$ is isomorphic over $B$ to $\ZZ\times B$.
+
+To start with, we observe that by the univalence axiom, $\UU$ is an object classifier, in the sense that $\sE(A,\UU)$ is naturally equivalent, by pullback of the canonical map $\UUp\to \UU$, to a full sub-$\infty$-groupoid of the core of $\sE/A$ (the fiberwise ``small'' objects).
+This means that for any $f,g:A\to \UU$, the induced map from the space of homotopies $f\sim g$ to the space of equivalences $f^*\UUp \equiv g^*\UUp$ in $\sE/A$ is an equivalence.
+This holds because the former is the space of lifts of $(f,g):A\to \UU\times \UU$ to the path-object $P\UU$, while the latter is (e.g.\ by~\cite[Lemma 4.3]{shulman:elreedy}) the space of sections of $\mathrm{Equiv}_A(f^*\UUp,g^*\UUp)$, or equivalently the lifts of $(f,g)$ to $\mathrm{Equiv}_{\UU\times \UU}(\pi_1^*\UUp,\pi_2^*\UUp)$, and the univalence axiom says precisely that $P\UU$ is equivalent to $\mathrm{Equiv}_{\UU\times \UU}(\pi_1^*\UUp,\pi_2^*\UUp)$ over $\UU\times\UU$.
+
+Similarly, $\sum_{X:\UU} (X\to X)$ is the exponential in $\sE/\UU$ of $\UU_p$ by itself, so by the pullback-stability and universal property of exponentials, the space of lifts of $f:A\to \UU$ to $\sum_{X:\UU} (X\to X)$ is equivalent to the space of endomorphisms of the corresponding object $f^*\UUp$ of $\sE/A$.
+Thus, $\sum_{X:\UU} (X\to X)$ classifies objects equipped with an endomorphism, in that $\sE(A, \sum_{X:\UU} (X\to X))$ is naturally equivalent to the $\infty$-groupoid of small objects of $\sE/A$ equipped with an endomorphism.
+
+Finally, we need to consider the topos-theoretic interpretation of propositional truncation.
+We can ignore its particular construction in {\UniMath} and focus on its universal property, which says that it is a reflection into propositions, i.e.\ for any proposition $P$ we have $(\Trunc{X}\to P) \equiv (X\to P)$.
+Interpreted fiberwise in an $(\infty,1)$-topos, this says that if $X\to A$ is any map then $\Trunc{X}\to A$ is a reflection into the sub-$(\infty,1)$-category of $\sE/A$ consisting of the monomorphisms, i.e.\ maps $P\to A$ whose diagonal $P\to P\times_A P$ is an equivalence.
```

Comment:
Re: "$\Trunc{X}\to A$ is a reflection"

I think you mean that the functor that sends $X\to A$ to $\Trunc{X}\to A$ is a reflection, etc.

Reply:
How about "$\Trunc{X}\to A$ is _its_ reflection"?

Reply:
That makes it sound like reflections are always unique.

Reply:
They are!

Reply:
Well, then, that's fine.

Reply:
Ah, somehow I was thinking of "retraction" instead.